Checkout/Checkout/Source/Model/Country.swift

```diff
@@ -277,6 +277,15 @@ public struct Country: Equatable {
     ]
   }
 
+    public init?(dialingCode: String) {
+        var dialingCode = dialingCode.components(separatedBy: .decimalDigits.inverted).joined()
```

Comment:
Why is this needed?

Reply:
For country code validation, we need to know `iso3166Alpha2` using `dialingCode`